infrastructure/terraform/ssm.tf

```diff
@@ -0,0 +1,556 @@
+    git-origin      = var.git-origin
+  }
+}
+
+resource "aws_ssm_parameter" "kms_key" {
+  name        = "/qppar-sf/${var.environment}/conversion_tool/KMS_KEY"
+  description = "KMS_KEY"
+  type        = "String"
+  value       = "arn:aws:kms:us-east-1:003384571330:alias/qpp-qrda3converter-dev-kms_alias"
+  overwrite   = true
+
+  lifecycle {
+    ignore_changes = [
+      value
+    ]
+  }
+  tags = {
+    Name            = "${var.project_name}-ssm-${var.environment}",
+    owner           = var.owner,
+    project         = var.project_name
+    terraform       = "true"
+    pagerduty-email = var.pagerduty_email
+    application     = var.application
+    sensitivity     = var.sensitivity
+    git-origin      = var.git-origin
+  }
+}
+
+resource "aws_ssm_parameter" "newrelic_api_key" {
+  name        = "/qppar-sf/${var.environment}/conversion_tool/NEWRELIC_API_KEY"
+  description = "NEWRELIC_API_KEY"
+  type        = "SecureString"
+  value       = "secret"
+  overwrite   = true
+
+  lifecycle {
+    ignore_changes = [
+      value
+    ]
+  }
+  tags = {
+    Name            = "${var.project_name}-ssm-${var.environment}",
+    owner           = var.owner,
+    project         = var.project_name
+    terraform       = "true"
+    pagerduty-email = var.pagerduty_email
+    application     = var.application
+    sensitivity     = var.sensitivity
+    git-origin      = var.git-origin
+  }
+}
+
+resource "aws_ssm_parameter" "nexus_creds" {
+  name        = "/qppar-sf/${var.environment}/conversion_tool/NEXUS_CREDS"
+  description = "NEXUS_CREDS"
+  type        = "SecureString"
+  value       = "secret"
+  overwrite   = true
+
+  lifecycle {
+    ignore_changes = [
+      value
+    ]
+  }
+  tags = {
+    Name            = "${var.project_name}-ssm-${var.environment}",
+    owner           = var.owner,
+    project         = var.project_name
+    terraform       = "true"
+    pagerduty-email = var.pagerduty_email
+    application     = var.application
+    sensitivity     = var.sensitivity
+    git-origin      = var.git-origin
+  }
+}
+
+resource "aws_ssm_parameter" "nexus_host" {
+  name        = "/qppar-sf/${var.environment}/conversion_tool/NEXUS_HOST"
+  description = "NEXUS_HOST"
+  type        = "String"
+  value       = "ec2-52-201-239-45.compute-1.amazonaws.com"
+  overwrite   = true
+
+  lifecycle {
+    ignore_changes = [
+      value
+    ]
+  }
+  tags = {
+    Name            = "${var.project_name}-ssm-${var.environment}",
+    owner           = var.owner,
+    project         = var.project_name
+    terraform       = "true"
+    pagerduty-email = var.pagerduty_email
+    application     = var.application
+    sensitivity     = var.sensitivity
+    git-origin      = var.git-origin
+  }
+}
+
+resource "aws_ssm_parameter" "org_name" {
+  name        = "/qppar-sf/${var.environment}/conversion_tool/ORG_NAME"
+  description = "ORG_NAME"
+  type        = "String"
+  value       = "cpc-plus-conversion-tool"
```

Comment:
We'll have to see what token organization name they used for this value. It's been changed between environments as well which may break some endpoints that are not required now so we can put this issue  off for a little later.

Reply:
The current values were copied out of the S3 bucket

Reply:
Yeah its no problem, but I think the last time they made the testing tokens, they labelled some org names with test in the value.